tests/app/main/test_suppliers.py

```diff
@@ -1639,8 +1624,9 @@ def test_should_not_allow_contact_details_without_phone_number(self):
     def test_should_not_allow_contact_details_with_invalid_phone_number(self):
         twentyone = "a" * 21
         res = self.client.post(
-            "/suppliers/company-contact-details",
+            "/suppliers/company-details",
             data={
+                'company_name': "My Company",
                 'contact_name': "Name",
                 'email_address': "name@email.com",
                 'phone_number': twentyone
```

Comment:
We don't class an alpha string as 'invalid'?

We should, probably - but if we are going to leave it as-is, can we rename the test to just say that we mean it can't be over 20 chars.

Reply:
There's no way to reliably validate a phone number, people add spaces, dashes, plus symbols, etc.  And alphabetical can also work - phone keys have letters on them, and my bank used to advertise it's phone number as something like `0800-THEBANK`.

Reply:
Yeah that's a fair point. I guess they can always change it later and they have to check/confirm it.